env/includes/settings/apps/tk-multi-breakdown.yml

```diff
@@ -14,6 +14,10 @@ includes:
 - ../../app_locations.yml
 
 ################################################################################
+settings.tk-multi-breakdown.nukestudio:
+  hook_scene_operations: '{self}/{engine_name}_scene_operations.py'
+  location: "@apps.tk-multi-breakdown.location"
+
 settings.tk-multi-breakdown:
```

Comment:
This is really similar to `settings.tk-multi-breakdown.houdini`. Perhaps make it `settings.tk-multi-breakdown.config` and reuse it for whichever engines have overrides?
Also, this needs to be removed from tk-nuke.yml (which does not have a scene breakdown hook override in sgtk_config)

Reply:
Yup, agreed +1